### PR TITLE
Dockerfile: step 2 towards non-writable FS

### PR DESCRIPTION
**Pull request by bryanlatten**
- Moved additional .ini vars to env versions
- Removed cont-init.d setters
- Changed test compose file to be different value than maintainer

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -109,6 +112,8 @@ RUN sed -i "s/listen = .*/listen = 127.0.0.1:9000/" $CONF_FPMPOOL && \
     sed -i "s/newrelic.daemon.logfile = .*/newrelic.daemon.logfile = \"\/dev\/stdout\"/" $CONF_PHPMODS/newrelic.ini && \
     sed -i "s/;newrelic.loglevel = .*/newrelic.loglevel = \"warning\"/" $CONF_PHPMODS/newrelic.ini && \
     sed -i "s/;newrelic.daemon.loglevel = .*/newrelic.daemon.loglevel = \"warning\"/" $CONF_PHPMODS/newrelic.ini && \
+    # Match FPM timeout directive with .ini max execution time
+    sed -i "s/;*request_terminate_timeout = .*/request_terminate_timeout = \${PHP_FPM_MAX_EXECUTION_TIME}/" $CONF_FPMPOOL && \
```
Comment (bryanlatten) [inline Dockerfile:116]: variable was actually being written to wrong location, actually exists in pool config - corrected!

File: container/root/etc/cont-init.d/09-php-fpm.sh [hunk 2]
```diff
@@ -1,25 +0,0 @@
-#!/usr/bin/with-contenv bash
```
Comment (bryanlatten) [inline container/root/etc/cont-init.d/09-php-fpm.sh:1]: all vars in here converted to full environment var representation, runtime config `sed`'ing no longer required

**Comment by jimdelois** [comment]
👍

File: docker-compose.yml [hunk 3]
```diff
@@ -5,7 +5,7 @@ ubuntu:
   environment:
     CFG_APP_DEBUG: 1
     SERVER_LOG_MINIMAL: 1
-    PHP_FPM_MEMORY_LIMIT: 256M
+    PHP_FPM_MEMORY_LIMIT: 257M
```
Comment (ingluisjimenez) [inline docker-compose.yml:8]: why the +1 ? to show that the variables are in fact used?

**Comment by ingluisjimenez** [comment]
👍

File: docker-compose.yml (hunk 3 above)
Comment (bryanlatten) [reply docker-compose.yml:8]: @ingluisjimenez correct

**Pull request merged**